code/src/components/PerfectJob.js

```diff
@@ -0,0 +1,34 @@
+import React from 'react';
+
+const perfectJobOptions = [
+  'Model (runway)',
+  'Wizard',
+  'Therapist'
+]
+
+export const defaultPerfectJobOption = perfectJobOptions[0]
+
+const PerfectJob = ({ perfectJob, setPerfectJob }) => {
+  const handlePerfectJobChange = (event) => {
+    setPerfectJob(event.target.value);
+  }
+
+  return (
+    <>
+      <p>If Borzois could have human jobs, the most perfect one would be:</p>
+      {perfectJobOptions.map((option) => (
+        <label htmlFor={`perfectJob${option}`} key={option} className="jobthing">
+          <input
+            id={`perfectJob${option}`}
+            type="radio"
+            value={option}
+            onChange={handlePerfectJobChange}
```

Comment:
When this question appears, there is a default chosen option (Model, runway). I don't really know what is the settings for this, but if i compare with mine I have my "event.target.value" connected to the "onchange" instead of the props.

Reply:
I handle that on line 13 🙂